website/docs/d/api_management.html.markdown

```diff
@@ -49,6 +49,8 @@ output "api_management_id" {
 
 * `portal_url` - The URL of the Publisher Portal.
 
+* `public_ip_adresses` - The Public IP adresses of the API Management Service.
```

Comment:
```suggestion
* `public_ip_adresses` - The Public IP addresses of the API Management Service.
```

Reply:
Corrected the mistake in the latest commit